<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ISISDiagnostics</class>
 <widget class="QWidget" name="ISISDiagnostics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>448</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_25">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="loInputFiles">
        <item>
         <widget class="MantidQt::API::MWRunFiles" name="dsInputFiles" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>41</verstretch>
           </sizepolicy>
          </property>
          <property name="findRunFiles" stdset="0">
           <bool>true</bool>
          </property>
          <property name="label" stdset="0">
           <string>Input Files</string>
          </property>
          <property name="multipleFiles" stdset="0">
           <bool>true</bool>
          </property>
          <property name="fileExtensions" stdset="0">
           <stringlist>
            <string>.raw</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="loCalibration">
        <item>
         <widget class="QCheckBox" name="ckUseCalibration">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Use Calibration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MantidQt::MantidWidgets::DataSelector" name="dsCalibration" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoLoad" stdset="0">
           <bool>true</bool>
          </property>
          <property name="loadLabelText" stdset="0">
           <string/>
          </property>
          <property name="workspaceSuffixes" stdset="0">
           <stringlist>
            <string>_calib</string>
           </stringlist>
          </property>
          <property name="fileBrowserSuffixes" stdset="0">
           <stringlist>
            <string>_calib.nxs</string>
           </stringlist>
          </property>
          <property name="showLoad" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbTimeSlice">
     <property name="title">
      <string>Time Slice</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="properties"/>
      </item>
      <item>
       <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppRawPlot" native="true">
        <property name="canvasColour" stdset="0">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPreview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppSlicePreview" native="true">
        <property name="canvasColour" stdset="0">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pbPlot">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Plot Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::API::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/MWRunFiles.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ckUseCalibration</tabstop>
  <tabstop>pbPlot</tabstop>
  <tabstop>pbSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
